app/celery/celery.py

```diff
@@ -47,3 +47,13 @@ def init_app(self, app):
         )
 
         self.conf.update(app.config)
+
+class NotifyCelerySMS(Celery):
```

Comment:
I assume here is where I'll somehow tell it to hook into the other worker, and define which queues I want?